pyproject.toml

```diff
@@ -27,15 +27,16 @@ license = "MIT"
 name = "scikeras"
 readme = "README.md"
 repository = "https://github.com/adriangb/scikeras"
-version = "0.11.0"
+version = "0.11.1"
 
 [tool.poetry.dependencies]
 importlib-metadata = {version = ">=3", python = "<3.8"}
 python = ">=3.8.0,<3.12.0"
 scikit-learn = ">=1.0.0"
 packaging = ">=0.21"
-tensorflow = {version = ">=2.11.0", optional = true}
-tensorflow-cpu = {version = ">=2.11.0", optional = true}
+tensorflow = {version = "<=2.12.1", optional = true}
+tensorflow-cpu = {version = "<=2.12.1", optional = true}
```

Comment:
Why `2.12.1`? Can we be compatible with any `2.12.X` version at least?

Reply:
Done ✅

Reply:
@adriangb